CHANGES.rst

```diff
@@ -4,8 +4,11 @@ Changes
 Version 0.9.0 (UNRELEASED)
 --------------------------
 
+- Adds new option ``--show-deleted-runs`` to ``list`` command to allow to display deleted workflows.
```

Comment:
Minor: WDYT about this suggestion? IMO it's more in line with other statements, meaning that command goes first, then the option
```
- Changes ``list`` to hide deleted workflows by default.
- Changes ``list`` command to allow displaying deleted workflows via ``--all`` and ``--show-deleted-runs`` options.
```